substrate-archive/src/database/models.rs

```diff
@@ -57,6 +58,8 @@ impl BlockModel {
 		let header = <B::Header as HeaderT>::new(block_num, extrinsics_root, state_root, parent_hash, digest);
 
 		let spec = self.spec as u32;
+		// do I need to include justifications here?
```

Comment:
I'd say they're not necessary here, because we don't need the justification to build the block / execute the block later on.